packages/client/hmi-client/src/utils/map-project-assets.ts

```diff
@@ -29,7 +29,9 @@ export const generateProjectAssetsMap = (searchAsset: string): ProjectAssetItems
 					assetId: asset.assetId.toString(),
 					assetName:
 						asset.assetName ??
-						`${asset.assetId.substring(0, 3)} - ${formatShort(asset?.updatedOn)}`,
+						`${asset.assetId.substring(0, 3)} - ${formatShort(
+							asset?.updatedOn ?? asset?.createdOn
+						)}`,
```

Comment:
this shouldn't be needed anymore - but added createdOn as a backup just in case

Reply:
Smart!!!